common/app/implicits/Strings.scala

```diff
@@ -46,11 +46,12 @@ trait Strings {
     }
   }
 
+  // Note, this method encodes parameter values so you don't have to
```

Comment:
😆 technically it encodes UriBuilder substitutions i.e. { and } so you don't have to, and the rest of the url characters so UriBuilder doesn't have to